Gazebo/README.md

```diff
@@ -6,4 +6,19 @@ has_children: true
 has_toc: false
 ---
 
-# TODO: Texto indicando os guias dessa pasta
+# Sobre o Gazebo
+
+Bem vindos ao indice do  **Gazebo!**
+
+Nesta seção, vamos introduzir para vocês o que exatamente é o ambiente de simulação incorporado do ROS e por que fazemos uso dele. É essencial você se familiarizar com o programa já que vai ser usado durante todo o restante do projeto.
+
+Se você ainda não sabe muito bem o que é ROS, suas funcionalidades e pontos positivos, recomendamos ler a **[seção do ROS](../Gazebo/README.md)**. Caso você já saiba o que é o ROS e o Gazebo, já tenha todos os programas que precisa instalados mas não sabe executar os projetos com essas novas ferramentas -- não se preocupe! Basta dar uma lida na **[seção de como executar os projetos](../HowToRun/README.md)** que você vai se virar, sem problemas.
+  
+---
+
+### **[O que é?](./WhatIs.md)**
+Antes de trabalhar no projeto da disciplina é importante você saber o que está usando. Para isso, dê uma olhanada no texto para aprender **[extamente o que é o Gazebo.](./WhyToUse.md)**
+
+### **[Por que usar?](./WhyToUse.md)**
```

Comment:
Acabei de notar que já que eu fiz o indíce baseado na `main`, em que o nome do arquivo de "Por que usar ferramentas de simulação?" é  `WhyToUse.md`, enquanto que o nome na branch `feature/why-to-use-simulation` é `WhyToUseSimulationTools.md` Devo mudar o link do índice para ficar com esse nome da branch ou deixo do jeito que tá?

Reply:
Deixa do jeito que está, a gente arruma isso na outra branch